uw-frame-components/portal/widgets/controllers.js

```diff
@@ -149,7 +149,21 @@ define(['angular'], function(angular) {
      */
     $scope.getPrettyDate = function(dateString) {
       // Create a new date if a date string was provided, otherwise return null
-      return dateString ? new Date(dateString) : null;
+      if (dateString === undefined) {
```

Comment:
I bring it up cause now that we've added the getPublishedDate stuff in the rssToJson, it's likely an item will come back with a pubDate of null